src/config/locales/en-US.json

```diff
@@ -1031,7 +1031,12 @@
     "open": "Open URL",
     "detected_url": "Detected URL",
     "unsupported_alert_title": "Unsupported URL!",
-    "unsupported_alert_desc": "Please scan a valid ecency url."
+    "unsupported_alert_desc": "Please scan a valid ecency url.",
+    "confirmTransaction": "Confirm transaction",
+    "approve": "Approve",
+    "cancel": "Cancel",
+    "multi_array_ops_alert": "Multiple operations array detected!",
+    "multi_array_ops_aler_desct": "Ecency currently do not multiple operations array"
```

Comment:
@aliseyalvi, @noumantahir This should probably say `Ecency does not support signing multiple operations, yet`